src/components/hint/index.tsx

```diff
@@ -195,13 +240,13 @@ class Hint extends BaseComponent {
   }
 
   get hintOffset() {
-    const {offset} = this.getThemeProps();
-    return offset || DEFAULT_HINT_OFFSET;
+    const {offset} = getThemeProps(this.props, this.context);
+    return (offset || DEFAULT_HINT_OFFSET) as number;
```

Comment:
Why the `as Number`?

Reply:
`getThemeProps` returns `any`, however I'll remove this, as it's not needed if we use `this.props`